omnibus/config/software/datadog-agent.rb

```diff
@@ -99,7 +99,7 @@
 
   # move around bin and config files
   move 'bin/agent/dist/datadog.yaml', "#{conf_dir}/datadog.yaml.example"
-  move 'bin/agent/dist/system-probe.yaml', "#{conf_dir}/system-probe.yaml.example"
+  move 'bin/agent/dist/system-probe.yaml', "#{conf_dir}/system-probe.yaml.example" unless windows_arch_i386?
```

Comment:
With only this condition, I believe the `move` would raise an exception when `WINDOWS_DDNPM_DRIVER` is not set (as `bin/agent/dist/system-probe.yaml` won't exist).